### ISSUE TITLE
Audio of video clip auto-starts when app is resumed

### ISSUE THREAD

**Issue by npark-mozilla** [mozilla-mobile/firefox-tv#2365]
## Steps to reproduce
- Go to Youtube, play a video
- Pause video, go to menu overlay
- Press home button to exit app
- Enter Firefox-TV app again

### Expected behavior
- Shows overlay, no video

### Actual behavior
- Shows overlay, but the audio of the previously selected video starts playing, this can be reproduced every time.  It is similar to https://github.com/mozilla-mobile/firefox-tv/issues/2245, but easier to run into for an average user

### Device information
* Fire TV device: 4K, Cube
* Latest affected Firefox version: Tip of the master
* Earliest affected Firefox version: ? 3.10-LAT3

**Comment by mcomella** [comment]
I'm guessing YouTube reload the page and starts to autoplay the current video.

I don't remember how the code worked but this could be a regression because we may have previously hid/removed the WebRenderFragment when the OverlayFragment is shown and now we might overlay the OverlayFragment over the WebRenderFragment

**Comment by npark-mozilla** [comment]
Yes it's a regression, it does not repro on our last release, but on the LAT from last week.

**Comment by npark-mozilla** [comment]
Also happens with non master-o as well.  In Cube, what happens is that upon resuming the app, the audio comes from Cube's internal speaker (which it never should), then shortly after it gets rerouted through HDMI output.
